File: PHP/Program/PetShop.php
<?php

class Petshop
{
    // Atribut-atribut yang akan dipakai pada main.php
    private $ID = 0;
    private $nama_produk = "";
    private $stok_produk = 0;
    private $harga_produk = 0.0;
    private $foto_produk = "";

    // CONSTRUCTOR dengan parameter
    public function __construct($ID = 0, $nama_produk = "", $stok_produk = 0, $harga_produk = 0.0, $foto_produk = "")
    {
        $this->ID = $ID;
        $this->nama_produk = $nama_produk;
        $this->stok_produk = $stok_produk;
        $this->harga_produk = $harga_produk;
        $this->foto_produk = $foto_produk;
    }

    // GETTER SETTER

    // Setter ID
    public function set_id($ID)
    {
        $this->ID = $ID;
    }

    // Getter ID
    public function get_id()
    {
        return $this->ID;
    }

    // Setter nama produk
    public function set_nama_produk($nama_produk)
    {
        $this->nama_produk = $nama_produk;
    }

    // Getter nama produk
    public function get_nama_produk()
    {
        return $this->nama_produk;
    }

    // Setter stok produk
    public function set_stokProduk($stok_produk)
    {
        $this->stok_produk = $stok_produk;
    }

    // Getter stok produk
    public function get_stok_produk()
    {
        return $this->stok_produk;
    }

    // Setter harga produk
    public function set_hargaProduk($harga_produk)
    {
        $this->harga_produk = $harga_produk;
    }

    // Getter harga produk
    public function get_harga_produk()
    {
        return $this->harga_produk;
    }

    // Setter foto produk
    public function set_fotoProduk($foto_produk)
    {
        $this->foto_produk = $foto_produk;
    }

    // Getter foto produk
    public function get_fotoProduk()
    {
        return $this->foto_produk;
    }

    // DESTRUKTOR
    public function __destruct()
    {
        
    }
}

File: PYTHON/Program/PetShop.py
class PetShop:

    # ATRIBUT-ATRIBUT YANG MAU DIINISIALISASIKAN
    __id = 0
    __nama_produk = ""
    __stok_produk = 0
    __harga_produk = 0.0

    # CONSTRUCTOR DENGAN PARAMETER UNTUK ME-NEW KAN
    def __init__(self, id = 0, namaProduk = "", stokProduk = 0, hargaProduk = 0.0):
        self.__id = id
        self.__nama_produk = namaProduk
        self.__stok_produk = stokProduk
        self.__harga_produk = hargaProduk

    # Getter
    def get_id(self): return self.__id
    def get_nama_produk(self): return self.__nama_produk
    def get_stok_produk(self): return self.__stok_produk
    def get_harga_produk(self): return self.__harga_produk

    # Setter
    def set_id(self, id): self.__id = id
    def set_nama_produk(self, namaProduk): self.__nama_produk = namaProduk
    def set_stok_produk(self, stokProduk): self.__stok_produk = stokProduk
    def set_harga_produk(self, hargaProduk): self.__harga_produk = hargaProduk

    # DESTRUKTOR
    # def __del__(self):
    #     print("contoh jika pakai destructor")